warehouse/templates/manage/project/manage_project_base.html

```diff
@@ -65,6 +65,18 @@
           {% trans %}Back to projects{% endtrans %}
         </a>
         <div class="vertical-tabs__content">
+          {% if project.lifecycle_status == "quarantine-enter" %}
+            <div class="callout-block callout-block--warning">
+              <p>{% trans %}This project has been quarantined.{% endtrans %}</p>
+              <p>
+                {% trans %}
+                  PyPI Admins need to review this project before it can be restored.
+                  While in quarantine, the project is not installable by clients,
+                  and cannot be being modified by its maintainers.
```

Comment:
Should we add a similar message as in help regarding contacting security@ here? Or link to the help page?

Reply:
A link to the help page makes sense, will add